pkgs/development/python-modules/patiencediff/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "patiencediff";
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0nm2242xgg59l06m54rzxp41aly3lxjh0a1s7h6dk7ryxjh002lv";
+  };
+
+  # too complicated to setup
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "This package contains the implementation of the patiencediff algorithm";
+    homepage = "https://www.breezy-vcs.org/";
+  };
```

Comment:
Could you add the license and a maintainer ?

Reply:
Sure thing!

Reply:
I hope it is fine that I added myself as maintainer. I did not want to push it to someone else.